### DISCUSSION TITLE
how to trigger a command from launch.json

### DISCUSSION THREAD

**Discussion by nsharma1989** [eclipse-theia/theia#8343] [General]
Hi In vscode we can trigger a command from launch.json like
`
"configurations": [
{
// ...
"name": "COMMAND_NAME", // The name of the command defined in package.json
// ...
}
`

but in theia the same isn't working. Any solutions?

**Comment by akosyakov** [comment]
It should be something like `${command:extension.pickNodeProcess}`.
